HomeWorks/src/main/java/com/epam/spring/homework2/beans/BeanB.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.spring.homework2.beans;
+
+import com.epam.spring.homework2.beans.validator.FieldsValidator;
+import com.epam.spring.homework2.beans.validator.MyValidator;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
`@Component` is not needed - according to item No 6 beans (BeanB, BeanC, BeanD) should be instantiated via configuration class with `@Bean` methods